test/RemoteExecutorConsoleApp/RemoteExecutorConsoleApp.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <StartupObject>RemoteExecutorConsoleApp.Program</StartupObject>
```

Comment:
?? Is this necessary?

Reply:
yes otherwise it gives an error of having multiple entry points. (I was not able to find the reason behind it though)

Reply:
Oh, I know why. Because it is in the `test` folder, and we are automatically referencing the `Test` SDK.

https://github.com/dotnet/machinelearning/blob/082e3fcd1df0a9568981a6d12ab529c11007a6ac/test/Directory.Build.props#L29

We could maybe remove that reference in this .csproj, or turn off the Target that is generating the other `Main` method.

Reply:
See

https://github.com/Microsoft/vstest/blob/069c5f8ef14caff221951d0807485df5d57f0f87/src/package/nuspec/netcoreapp/Microsoft.NET.Test.Sdk.targets#L29-L39

for the parameter that turns this off